README.md

```diff
@@ -1,11 +1,133 @@
+
+#### Additional setup
+
+Before installing this extension, you'll need to:
+
+- [Set up Cloud Firestore in your Firebase project.](https://firebase.google.com/docs/firestore/quickstart)
+- Run Meilisearch instance. There are many easy ways [to download and run a Meilisearch instance.](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)
+
+#### Data import format
+
+Documents indexed in Meilisearch need to have a [unique id](https://docs.meilisearch.com/learn/core_concepts/documents.html#primary-field), to do so this extension will use the default field provided for this purpose in Firestore: `Document ID` which will be renamed `_firestore_id` used as [document id](https://docs.meilisearch.com/learn/core_concepts/documents.html#document-id). If your documents have an `id` field it will be added to your documents without it being your primary key.
+
+**Important:**  If your document have a field `_firestore_id` it will be ignored.
+
+[GeoSearch](https://docs.meilisearch.com/reference/features/geosearch.html#geosearch) has a specific format in Meilisearch, if a `GeoPoint` from Firestore with the name `_geo` is found the field `latitude` would renamed to `lat` and the field `longitude` to `lng`.
+If a `GeoPoint` is found without the name `_geo` it would added like an array.
+
+#### Backfill your Meilisearch
+
+This extension will only export your documents if they have been modified or created after its installation -- it does not export your complete dataset of existing documents into Meilisearch. So, to backfill your Meilisearch dataset with all the documents present in your Firestore collection, you can run the [import script](https://github.com/meilisearch/firestore-meilisearch/) provided by this extension.
+
+#### Billing
+
+To install an extension, your project must be on the [Blaze (pay as you go) plan](https://firebase.google.com/pricing)
+
+* You will be charged a small amount (typically around $0.01/month) for the Firebase resources required by this extension (even if it is not used).
+* This extension uses other Firebase or Google Cloud services which may have
+  associated charges if you exceed the service’s free tier:
+  *   Cloud Firestore
+  *   Cloud Functions (Node.js 10+ runtime. [See FAQs](https://firebase.google.com/support/faq#extensions-pricing))
+
+When you use Firebase Extensions, you're only charged for the underlying
+resources that you use. A paid-tier billing plan is only required if the
+extension uses a service that requires a paid-tier plan, for example calling to
+a Google Cloud API or making outbound network requests to non-Google services.
+All Firebase services offer a free tier of usage.
+[Learn more about Firebase billing.](https://firebase.google.com/pricing)
+
+
+
+
+**Configuration Parameters:**
```

Comment:
The `README.md` file is auto-generated by the `firebase CLI` (a tool for managing, viewing, and deploying to Firebase projects) from the `PREINSTALL.md`, `POSTINSTALL.md` and `extension.yaml` documents. So all the page layouts from line 26 to 100 
The `firebase CLI` (a tool to manage, view and deploy Firebase projects) allows to auto-generate the `README.md` file from the `PREINSTALL.md`, `POSTINSTALL.md` and `extension.yaml` documents. It why there is a lot of repeating sentences.
I used this tool to generate the text from lines 26 to 100, so all the page layout came from this tool. I decided to use this tool because it allows consistency in the `README` with other firebase projects